EIPS/eip-1845.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: <to be assigned>
+title: Fork Name Standards
+author: DimitryKh <dimitry@ethereum.org>
+discussions-to: https://github.com/ethereum/retesteth/issues/23
+status: Draft
+category: Informational
+created: 2019-03-18
+---
+
+## Simple Summary
+Name forks like  `ETH-001-00, ETH-002-00, ETH-003-00` and so on.
+
+## Abstract
+Each client parses `ETH-001-00` kind of indexes into ethereum fork rules, the major number increasing for roadmap forks and the minor for other forks. `ETH-001-00` stands for Frontier rules.
+Therefore if in any config file instead of `Frontier`, an `ETH-001-00` name provided. The client must parse it as a `Frontier` rule.
+Parsing previous naming convention (`Frontier, Homestead, EIP150`) becomes optional. `ETH-xxx-yy` naming convention is mandatory for all clients to implement. 
+
+## Motivation
+**Clients:**  
+Different clients use different fork names even for the main network on their genesis config. If we want a generic genesis format, we have to use a standard for the fork names. 
```

Comment:
That's correctly identifies an issue, but do not explain why you need to create new standard instead of using the existing "standard" of "Hardfork Meta" EIPs: https://eips.ethereum.org/meta.

Reply:
Name your pets, number your herd.  This article is about servers but the intent is the same: https://www.engineyard.com/blog/pets-vs-cattle If we are going to go to 2-4 upgrades a year we don't need to spend time bikeshedding fork block names and waiting on someone to make it "official."